src/building/how-to-build-and-run.md

```diff
@@ -1,10 +1,7 @@
 # How to Build and Run the Compiler
 
 The compiler is built using a tool called `x.py`. You will need to
-have Python installed to run it. But before we get to that, if you're going to
-be hacking on `rustc`, you'll want to tweak the configuration of the compiler.
-The default configuration is oriented towards running the compiler as a user,
-not a developer.
+have Python installed to run it.
```

Comment:
Was it intentional to remove this bit?

Reply:
Yes, although let me know if it's too cheeky to sneak it in here rather than making an entirely separate PR for it. The lines that were removed were directly contradictory to the rest of the document; you don't want to tweak configuration *before* you install Python, you want to install Python so that you can run `./x.py setup` so that an appropriate config can be generated for you (and there is no "default" configuration, the setup tool prompts you to indicate what you're doing, which also encompasses developer rather than user usecases).

Reply:
fair enough, LGTM